README.adoc

```diff
@@ -30,9 +30,9 @@ https://ci.centos.org/job/minishift/[image:https://ci.centos.org/buildStatus/ico
 To download the latest binary and review release notes, see
 the https://github.com/minishift/minishift/releases[Minishift releases] page.
 
-Minishift requires a hypervisor to start the virtual machine containing
-OpenShift. Make sure that the hypervisor of your choice is installed and
-enabled on your system before you install Minishift.
+Minishift requires a hypervisor to start the virtual machine that provisions OpenShift.
```

Comment:
This seems to say that the VM is provisioning OpenShift. That's not correct. 

"Minishift requires a hypervisor to start the virtual machine on/onto which OpenShift is provisioned"